opentelemetry-operator/chart/Chart.yaml

```
@@ -2,8 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 apiVersion: v2
+appVersion: 0.104.0
```

```suggestion
appVersion: v0.104.0
```

I'd name it exactly like the version upstream.